File: 283-move-zeroes/move-zeroes.cpp
class Solution {
public:
    void moveZeroes(vector<int>& nums) {
        int f=0;
        int n=nums.size();
        for (int i=0;i<n;i++)
        {
            if(nums[i]!=0){
                int t=nums[i];
                nums[i]=nums[f];
                nums[f]=t;
                f++;

            }
            cout<<nums[i];
        }
    }
};

File: 283-move-zeroes/move-zeroes.py
class Solution:
    def moveZeroes(self, num: List[int]) -> None:
        a=0
        for i in range(len(num)):
            if num[i]!=0:
                num[a]=num[i]
                a+=1
        for i in range(a,len(num)):
            num[i]=0
        print(num)

        """
        Do not return anything, modify nums in-place instead.
        """
        